test/functional/commands/jetpack-compose-e2e-specs.js

```diff
@@ -17,6 +17,11 @@ describe('Jetpack Compose', function () {
     }
     driver = await initSession(COMPOSE_CAPS);
   });
+
+  afterEach(async function () {
+    await driver.back();
```

Comment:
('m not quite sure this is going to work. `afterEach` is executed if any on the test steps fail. What if, for example, 
```
   let el = await driver.elementByXPath("//*[@text='Clickable Component']");
    await driver.moveTo(el);
    await el.click();
```
one of these steps fail? Then returning back would redirect us to a home screen.

If we use afterEach then beforeEach should also have appropriate content

Reply:
Yes, I borrowed it from `mobile-e2e-specs.js`. I could not find any example with `finally` in any of tests. Having interdependent tests is bad and probably we should switch to creating driver in before each and destroying it in after each.